### PR TITLE
Add debugging support to the list of LSP features

### PR DESCRIPTION
**Pull request by rafaelfranca**
Is there any more information that we need to include. For example how to add the launch.json config?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,7 @@ The Ruby LSP features include
 - Format on save (with RuboCop or Syntax Tree)
 - Format on type
 - Require path completion
+- Debugging support
```
Comment (vinistock) [inline README.md:118]:
```suggestion
- Debugging support
- Running and debugging tests through VS Code's UI
```

File: package.json [hunk 2]
```diff
@@ -218,7 +218,7 @@
         },
         "configurationSnippets": [
           {
-            "label": "Debug a Ruby program",
+            "label": "Ruby: Debug program",
```
Comment (rafaelfranca) [inline package.json:221]: I changed those to follow the conventions of other languages:

Screenshot 2023-06-28 at 5 18 47 PM

**Review by vinistock — APPROVED** [review]
Nice. Thanks for updating our docs!

**Comment by vinistock** [comment]
For the formatting errors, `yarn run format` should do it.

**Pull request merged**